vscode/build.gradle

```
@@ -0,0 +1,111 @@
+import org.apache.tools.ant.taskdefs.condition.Os
+import groovy.json.JsonSlurper
+
+plugins {
+  id "com.github.node-gradle.node" version "2.2.0"
+}
+
+apply plugin: 'com.github.node-gradle.node'
+
+def packageSlurper = new JsonSlurper()
+def packageJson = packageSlurper.parse file('package.json')
+version = packageJson.version
+
+def vscePath = './node_modules/vsce/out/vsce'
+
+node {
+    version = '10.14.1'
+    npmVersion = '6.4.1'
+    download = true
+}
+
+
+// npm_run_compile {
+    // inputs.files fileTree(
+    //   dir: "src",
+    //   excludes: ["bin/**", "node_modules/**"])
+    // inputs.file 'package.json'
+    // inputs.file 'package-lock.json'
+    // inputs.file 'tsconfig.json'
+    // outputs.dir 'out'
+// }
+
+npmInstall {
+		inputs.files fileTree("./")
+}
+
+task copyLsp(type: Copy) {
+  from("$rootProject.projectDir/build/distribution/lsp")
```

```suggestion
  from("$rootDir/build/distribution/lsp")
```

use [rootDir](https://docs.gradle.org/current/javadoc/org/gradle/api/Project.html#getRootDir--)